src/background/services/outgoingPaymentGrant.ts

```diff
@@ -233,7 +233,7 @@ export class OutgoingPaymentGrantService {
     const nonce = crypto.randomUUID();
     try {
       const grant = await this.openPaymentsService.client.grant.request(
-        { url: walletAddress.authServer },
+        { url: ensureEnd(walletAddress.authServer, '/') },
```

Comment:
For v1.x, the end slash is necessary for hash verification, but not in the original grant request necessarily

Reply:
I added this `/` here to have same code for v1 and v2 during verification.